website/docs/intro/install.mdx

```diff
@@ -40,7 +40,7 @@ OpenTofu is available via [PackageCloud](https://packagecloud.io/opentofu/tofu)
 
 ### Debian and derivatives:
 ```bash
-curl -s https://packagecloud.io/install/repositories/opentofu/tofu/script.deb.sh -o /tmp/tofu-repository-setup.sh
+curl -s https://packagecloud.io/install/repositories/opentofu/tofu/script.deb.sh?any=true -o /tmp/tofu-repository-setup.sh
```

Comment:
**(info)** It appears the this query does not play a role - I tried both commands, no difference noticed 🤔

Reply:
It does make some slight changes on debian, I was running into errors:
```diff
141c141
<   apt_config_url="https://packagecloud.io/install/repositories/opentofu/tofu/config_file.list?os=any&dist=any&source=script"
---
>   apt_config_url="https://packagecloud.io/install/repositories/opentofu/tofu/config_file.list?os=${os}&dist=${dist}&source=script"
143c143
<   apt_source_path="/etc/apt/sources.list.d/opentofu_tofu_any.list"
---
>   apt_source_path="/etc/apt/sources.list.d/opentofu_tofu.list"
149c149
<     gpg_key_path_old="/etc/apt/trusted.gpg.d/opentofu_tofu_any.gpg"
---
>     gpg_key_path_old="/etc/apt/trusted.gpg.d/opentofu_tofu.gpg"
```
```diff
150c150
<   repo_config_name=opentofu_tofu_rpm_any
---
>   repo_config_name=opentofu_tofu
155c155
<   yum_repo_config_url="https://packagecloud.io/install/repositories/opentofu/tofu/config_file.repo?os=rpm_any&dist=rpm_any&source=script"
---
>   yum_repo_config_url="https://packagecloud.io/install/repositories/opentofu/tofu/config_file.repo?os=${os}&dist=${dist}&source=script"
210,213d209
<   yum_repo_base_path=`dirname ${yum_repo_path}`
< 
<   # modify config file contents to add 'rpm_any' after repo name
<   sed -i "s/opentofu_tofu/${repo_config_name}/g" $yum_repo_base_path/$repo_config_name.repo
```